tests/test_execute.py

```diff
@@ -191,7 +191,11 @@ def test_execute_notebook_save_profiling_data(tmp_empty):
     outfile = "out-profiling-data.csv"
     assert Path(outfile).is_file()
     with open(outfile, "r") as f:
-        lines = f.readlines()
+        read_lines = f.readlines()
```

Comment:
why was this breaking on windows? it'd be good to add a comment explaining why this is needed

Reply:
Added comments